### DISCUSSION TITLE
How to package custom plugin with eip format

### DISCUSSION THREAD

**Discussion by sekfung** [emqx/emqx#7437] [Q&A] [answered]
I want to install my custom plugin by uploading dashbord, and what I should do ?

![image](https://user-images.githubusercontent.com/43331641/160464461-8668157e-ad1d-4dda-b529-04ee6d2e5e88.png)

**Comment by zmstone** [comment, marked as answer]
Hi

you’d need to build a plugin package.
Please follow the example here: 

https://github.com/emqx/emqx-plugin-template

Btw. There is a typo in the UI.
Should be tar.gz format.
